disnake/ext/commands/ctx_menus_core.py

```diff
@@ -125,7 +125,7 @@ async def __call__(
         self, interaction: ApplicationCommandInteraction, target: Any = None, *args, **kwargs
     ) -> None:
         # the target may just not be passed in
-        args = (target or interaction.target,) + args
+        args = (target or interaction.target, *args)
```

Comment:
We should reconsider `RUF005`: https://github.com/charliermarsh/ruff/issues/2142#issuecomment-1402722345
While the changes themselves are generally fine, the speed increase is questionable and readability suffers sometimes.

Reply:
https://github.com/DisnakeDev/disnake/pull/935/commits/9885350a598bc314c3251c886ad797ea5c116815